app/pages/admin/user-settings/details.jsx

```diff
@@ -14,8 +20,50 @@ const UserDetails = (props) => {
         <li>Login: {props.user.login}</li>
         <li>Display name: {props.user.display_name}</li>
         <li>Email address: {props.user.email}</li>
+        <li>Signed up: {created.toString()}</li>
+        <li>
+          <label>
+            Valid email:{' '}
+            <input
+              type="checkbox"
+              name="valid_email"
+              checked={props.user.valid_email}
+              onChange={onChange}
+            />
+          </label>
+        </li>
       </ul>
 
+      <form>
+        <fieldset>
+          <legend>Email preferences</legend>
+          <ul>
+            <li>
+              <label>
+                <input
+                  type="checkbox"
+                  name="global_email_communication"
+                  checked={props.user.global_email_communication}
+                  onChange={onChange}
+                />{' '}
+                Zooniverse general emails
+              </label>
+            </li>
+              <label>
+                <input
+                  type="checkbox"
+                  name="beta_email_communication"
+                  checked={props.user.beta_email_communication}
+                  onChange={onChange}
+                />{' '}
+                Beta test emails
+              </label>
+            <li>
```

Comment:
should go on line 52 to wrap the `<label>`?

also, I'm not sure if we should add `<li>` and label/input for NASA email communication?

Reply:
Sorry, I'm not sure what Nasa email communication is. Is it something that admins need to set/reset on user accounts?

Reply:
I've added it to the form as 'NASA emails.'